### DISCUSSION TITLE
Changing the placeholder searchbar text

### DISCUSSION THREAD

**Discussion by Jurick-Li** [YannickRe/spfx-msgraph-peoplesearch#69] [Q&A] [answered]
Hi,

I would like to change the placeholder text in the searchbar on default it is in english but i want to change it to dutch, i have tried changing a few lines in mystrinds.d.ts and peoplesearchbox.tsx. Do you have any tips on how i could do this?

**Comment by YannickRe** [comment, marked as answer]
You can either change the value(s) in the en-us.js file: https://github.com/YannickRe/spfx-msgraph-peoplesearch/blob/master/src/webparts/peoplesearch/loc/en-us.js#L17
or you can create a nl-nl.js or nl-be.js file in that folder, as a copy of the en-us.js and then package and deploy.

**Reply by Jurick-Li** [reply]
Thanks, i totally overlooked that one!
